tests/POW/test_POW.cpp

```diff
@@ -740,6 +740,38 @@ BOOST_AUTO_TEST_CASE(mining_and_verification) {
       blockToUse, difficultyToUse, rand1, rand2, ipAddr, pubKey, 0, 0, false,
       winning_nonce, winning_result.result, winning_result.mix_hash);
   BOOST_REQUIRE(!verifyWinningNonce);
+
+  // Full Dataset mine and verify
+  winning_result = POWClient.PoWMine(blockToUse, difficultyToUse, rand1, rand2,
```

Comment:
Please change it to  winning_result = POWClient.PoWMine(blockToUse, difficultyToUse, rand1, rand2,
                                     ipAddr, pubKey, 0, 0, true);
The last parameter is fullDataset mining

Reply:
Thanks for pointing this out. After doublechecking it seems, that this needs to be corrected also in multiple PoWVerify calls in this test suite. I fixed that in commit dec3bdfe0772ad780d86a8a2ccf4c514a072aa56 but when enabling the OPENCL I still get stalled test. Does this work on your machine? I'm not sure if this problem is caused by my graphics card driver or not.

Reply:
Yes, the others i will handle in my pull request. You only need to fix this one.